src/utils/install/journal_defaults.json

```diff
@@ -2939,6 +2939,21 @@
             "default": "<p>You may upload a file in lieu of completing the review form. Please ensure you've covered all of the required fields below.</p>"
         }
     },
+    {
+        "group": {
+            "name": "general"
+        },
+        "setting": {
+            "description": "Enable open peer review mode on this journal.",
```

Comment:
We could use a better description here, explaining what this setting actually does